django/contrib/admin/options.py

```diff
@@ -2024,10 +2024,13 @@ def __init__(self, parent_model, admin_site):
         self.opts = self.model._meta
         self.has_registered_model = admin_site.is_registered(self.model)
         super().__init__()
+        if self.verbose_name_plural is None:
+            if self.verbose_name is None:
+                self.verbose_name_plural = self.model._meta.verbose_name_plural
+            else:
+                self.verbose_name_plural = self.verbose_name + 's'
```

Comment:
Make sure you use `format_lazy()` to prevent issues with translated strings:
```suggestion
                self.verbose_name_plural = format_lazy('{}s', self.verbose_name)
```
This is also consistent with the following: https://github.com/django/django/blob/c70cd2a926ffab47f6613e83e0c8828eb6c2c064/django/db/models/options.py#L188-L191

Reply:
Done. I actually wondered about using that, but did not know the reason why. Thank you for pointing that out.

Also took the opportunity to re-order the imports, because that failing isort test was annoying me and at least some of it was my fault.